src/um/processthreadsapi.rs

```diff
@@ -60,31 +68,32 @@ STRUCT!{struct STARTUPINFOW {
     hStdError: HANDLE,
 }}
 pub type LPSTARTUPINFOW = *mut STARTUPINFOW;
+
 extern "system" {
     pub fn QueueUserAPC(
         pfnAPC: PAPCFUNC,
         hThread: HANDLE,
-        dwData: ULONG_PTR,
+        dwData: ULONG_PTR
```

Comment:
https://github.com/retep998/winapi-rs/blob/dev/CONTRIBUTING.md#extern-functions has a trailing comma. Macros for functions (like `RIDL!` and `FN!`) *require* trailing commas. So yes, trailing commas for all functions.